leptos_macro/src/template.rs

```diff
@@ -387,9 +387,13 @@ fn child_to_tokens(
                 ))
             }
         }
+        //TODO Should we ewerywhere use syn::expr? 
```

Comment:
Could you say a little more about what you mean here?

Reply:
I am just wasn't sure what block_to_tokens is doing. And what it expect for different `syn::Expr`.

There also was `str_value match` that looks like `value_to_string` to string body.

So my thought about `block_to_tokens` was:
1. should it reuse `value_to_string`?
2. if it tries to handle primitive literals differently maybe this handling should be done on higher lever (split this function into two (text_to_tokens/dyn_block_to_tokens), and use them in place where needed?

Reply:
Ah okay. No, I think it was just legacy code. Yes `str_value` is same as `value_to_string`.